app/rss.py

```diff
@@ -0,0 +1,62 @@
+import xml.etree.ElementTree as ET
+
+from pathlib import Path
+from podgen import Podcast, Episode, Person, Media
+
+from flask import Response
+from app import app
+
+from . import models
+
+file = Path('app/templates/feed_template.xml')
+
+
+class RssResponse(Response):
+    def __init__(self, response, **kwargs):
+        if 'mimetype' not in kwargs and 'contenttype' not in kwargs:
+            if response.startswith('<?xml'):
```

Comment:
This is a very lousy way to guess XML content. You have already imported ElementTree, why not use it here? 